xml/obs/openSUSE:Leap:15.3:Images.xml

```diff
@@ -8,6 +8,12 @@
         <!-- Due to there being multiple archs, it omits "head -n1". So don't match .sha256(.asc). -->
         <hdd filemask=".*kvm-and-xen.*\.qcow2$"/>
     </flavor>
+    <flavor name="JeOS-for-AArch64" folder="*/JeOS:JeOS-efi.aarch64">
+        <hdd filemask=".*ARM-JeOS-efi.*\.raw\.xz"/>
```

Comment:
I missed this, it also picked up checksums:

```suggestion
        <hdd filemask=".*ARM-JeOS-efi.*\.raw\.xz$"/>
```

Reply:
Fixed!